docs/getting-started/03_build.md

```diff
@@ -56,6 +56,14 @@ You should also run `yarn dist` when you pull new work, to make sure you have bu
 We use [Composer](https://getcomposer.org/) to maintain PHP dependencies. This allows us to **not** version
 control stuff that is already version controlled elsewhere.
 
+### Installing Dependencies via Composer
+
+On host machine:
+`composer isntall`
```

Comment:
```suggestion
`composer install`
```
Same warning here...version should match the project's set in composer.json/composer.lock.